### PR TITLE
Remove mil_vision/great_merge_purgatory

### PR DESCRIPTION
**Pull request by DSsoto**
* Removes the great_merge_purgatory in mil_vision
  - Keeps camera_lidar_transformer
  - Code from pinger_finder is removed but then refactored into _PassiveSonarOS™_
* camera_stream_demo is moved from the nodes directory to the src dir.

### CODE REVIEW

**Review by kev-the-dev — APPROVED** [review]
Good

File: utils/mil_msgs/srv/CameraToLidarTransform.srv [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Header header #Stamp of time point was seen for tf
```
Comment (kev-the-dev) [inline utils/mil_msgs/srv/CameraToLidarTransform.srv:1]: Don't you need to add this service to cmake?

**Review by kev-the-dev — CHANGES_REQUESTED** [review]
Check this one thing out

**Comment by sentree** [comment]
This built on the CI and passed all unit tests except the ros_alarms test that always fails.
-- Your friendly temporary CI guy

**Pull request merged**